base/REPLCompletions.jl

```diff
@@ -291,7 +291,7 @@ function completions(string, pos)
     # First parse everything up to the current position
     partial = string[1:pos]
     inc_tag = Base.syntax_deprecation_warnings(false) do
-        Base.incomplete_tag(parse(partial, raise=false))
+        Base.incomplete_tag(Base.parse_input_line(partial))
```

Comment:
@Carreau It would be good if you also add your multi-line tests to cover some of the code paths that uses the `inc_tag`. This means it would be good to add your `multiline` function to these two test https://github.com/Carreau/julia/blob/mlcomplete/test/replcompletions.jl#L440-L448 for path completion and these two https://github.com/Carreau/julia/blob/mlcomplete/test/replcompletions.jl#L200-L215 for method completion.